deployments/puppet/templates/splunk-otel-collector.conf.erb

```diff
@@ -13,6 +13,7 @@ SPLUNK_LISTEN_INTERFACE=<%= @splunk_listen_interface %>
 <% end -%>
 SPLUNK_REALM=<%= @splunk_realm %>
 SPLUNK_TRACE_URL=<%= @splunk_trace_url %>
+GOMEMLIMIT=<%= @gomemlimit %>
```

Comment:
The `gomemlimit` parameter should be added to init.pp and documented in the readme.

Reply:
okay